shiny/global-sensitivity/helper.R

```diff
@@ -0,0 +1,136 @@
+library(shiny)
+library(PEcAn.DB)
+library(RPostgreSQL)
+library(udunits2)
+library(lubridate)
+library(dplyr)
+
+options(scipen=12)
+
+betyConnect <- function(php.config="../../web/config.php") {
```

Comment:
Agreed. I'll look into merging these into the current visualizations package, but if they're too shiny-specific, I might make a separate PEcAn.shiny package for them.